x-pack/plugins/security_solution/common/detection_engine/schemas/request/find_exception_list_references_schema.ts

```diff
@@ -9,13 +9,19 @@ import * as t from 'io-ts';
 import { NonEmptyStringArray } from '@kbn/securitysolution-io-ts-types';
 import { DefaultNamespaceArray } from '@kbn/securitysolution-io-ts-list-types';
 
-export const findExceptionReferencesOnRuleSchema = t.exact(
-  t.type({
-    ids: NonEmptyStringArray,
-    list_ids: NonEmptyStringArray,
-    namespace_types: DefaultNamespaceArray,
-  })
-);
+export const findExceptionReferencesOnRuleSchema = t.intersection([
+  t.exact(
+    t.type({
+      namespace_types: DefaultNamespaceArray,
+    })
+  ),
+  t.exact(
+    t.partial({
```

Comment:
tbh, it's unclear for me why `ids` and `list_ids` became optional

Reply:
I'll add some comments to make it clear. Without those fields we will just fetch _all_ lists that match the namespace type sent in.